src/core/or/circuitmux.c

```diff
@@ -981,18 +980,15 @@ circuitmux_make_circuit_active(circuitmux_t *cmux, circuit_t *circ)
  */
 
 static void
-circuitmux_make_circuit_inactive(circuitmux_t *cmux, circuit_t *circ)
+circuitmux_make_circuit_inactive(circuitmux_t *cmux, circuit_t *circ,
+                                 chanid_circid_muxinfo_t *hashent)
 {
   tor_assert(cmux);
   tor_assert(cmux->policy);
   tor_assert(circ);
 
   /* Policy-specific notification */
   if (cmux->policy->notify_circ_inactive) {
-    /* Okay, we need to check the circuit for policy data now */
-    chanid_circid_muxinfo_t *hashent = circuitmux_find_map_entry(cmux, circ);
-    /* We should have found something */
-    tor_assert(hashent);
```

Comment:
This assertion could stay -- and as before, it would be good to check this for consistency with cmux and circ.